dt-core/admin/components/setup-wizard.js

```diff
@@ -313,6 +322,10 @@ export class SetupWizard extends LitElement {
     const url = new URL(location.href);
 
     this.isKitchenSink = url.searchParams.has('kitchen-sink');
+    //get step number from step url param
+    if (url.searchParams.has('step')) {
+      this.currentStepNumber = parseInt(url.searchParams.get('step'));
+    }
```

Comment:
Is this for debugging?

Reply:
yes. makes working on a step a lot easier

Reply:
cool, thanks, good idea